flexmeasures/conftest.py

```diff
@@ -154,28 +148,32 @@ def create_test_accounts(db) -> dict[str, Account]:
         ],
     )
     db.session.add(multi_role_account)
-    return dict(
-        Prosumer=prosumer_account,
-        Supplier=supplier_account,
-        Dummy=dummy_account,
-        Multi=multi_role_account,
-    )
-
-
-def add_consult_account_access_to_prosumer_account(db, test_accounts):
-    account = Account.query.filter_by(name="Test Prosumer Account").one_or_none()
     consultant_account_role = AccountRole(
-        name="Consultant",
-        description="Account that prosumer can access",
+        name="Consultant", description="A Consultant Account"
     )
     consultant_account = Account(
-        name="Test Consultant Account",
-        account_roles=[consultant_account_role],
-        consultancy_account_id=account.id,
+        name="Test Consultant Account", account_roles=[consultant_account_role]
     )
     db.session.add(consultant_account)
+    consultant_client_account_role = AccountRole(
+        name="ConsultantClient",
+        description="A Client of a Consultant",
+    )
+    account = Account.query.filter_by(name="Test Consultant Account").one_or_none()
+    consultant_client_account = Account(
```

Comment:
I believe we don't need to add this account. We could give the consultant the PRosumer as a client, as we're not doing anything invasive anyway. Would also spare us another asset.

```
prosumer_account.consultancy_account_id = account.id
```

(maybe `account` is a variable name which can lead to confusions btw)

Reply:
I've removed the account variable in favor of just the id with a more descriptive name: `consultant_account_id = Account.query.filter_by(name="Test Consultant Account").one_or_none().id`

I think the extra consultant client account makes it easier to see the relationship for future reference. I think the Prosumer name can also be confusing for being a consultant client account.

I can change it to test on the Prosumer if needed but I'm not sure it's worth changing this in terms of resources or making the test setup more clear.